lib/index.js

```diff
@@ -17,14 +18,17 @@ var Tap = require('./tap');
 var internals = {};
 
 
-exports.agents = {
+module.exports = new Events.EventEmitter();
```

Comment:
This is kinda hackish. Better to use normal prototype style and new an object and export it.